### PR TITLE
Move `excludes` values to the `JunitJacocoExtension` configuration stage, to allow easier modification.

### PR DESCRIPTION
**Pull request by gtcompscientist**
This moves the setup of `excludes` as mentioned in #172

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 Version 0.17.0 *(In development)*
 ---------------------------------
+- Move `excludes` values to the `JunitJacocoExtension` configuration stage, to allow easier modification.
```
Comment (vanniktech) [inline CHANGELOG.md:5]: Can you revert the changes here? I'm generating this file when doing a new release.

Comment (gtcompscientist) [reply CHANGELOG.md]: Done! Also merged my commits to be cleaner.
